pipeline/transformer/transformer_mets/src/test/scala/uk/ac/wellcome/platform/transformer/mets/parsers/MetsXmlParserTest.scala

```diff
@@ -29,75 +29,64 @@ class MetsXmlParserTest extends FunSpec with Matchers {
     MetsXmlParser(xmlMultipleLicense) shouldBe a[Left[_, _]]
   }
 
-  it("fails if the input stream is not an xml") {
-    MetsXmlParser("hagdf".toInputStream) shouldBe a[Left[_, _]]
+  it("fails if the input string is not an xml") {
+    MetsXmlParser("hagdf") shouldBe a[Left[_, _]]
   }
 
-  def xml = getClass.getResourceAsStream("/b30246039.xml")
+  def xml =
+    IOUtils.toString(getClass.getResourceAsStream("/b30246039.xml"), "UTF-8")
 
   def xmlNodmdSec =
-    s"""
-       |<mets:mets xmlns:mets="http://www.loc.gov/METS/" xmlns:mods="http://www.loc.gov/mods/v3">
-       |  <mods:recordIdentifier source="gbv-ppn">b30246039</mods:recordIdentifier>
-       |</mets:mets>
-       |""".stripMargin.toInputStream
+    <mets:mets xmlns:mets="http://www.loc.gov/METS/" xmlns:mods="http://www.loc.gov/mods/v3">
```

Comment:
https://contributors.scala-lang.org/t/proposal-to-remove-xml-literals-from-the-language/2146/94
🤷‍♂

Reply:
I know, but I think it's an easy fix when we'll have to upgrade scala. In the meantime, it makes code a bit more maintainable and readable